Jenkinsfile

```diff
@@ -7,7 +7,7 @@ pipeline {
         sh 'yarn'
         sh 'yarn run flow'
         sh 'yarn run lint'
-        sh 'yarn run test --runInBand'
+        sh 'yarn run test --coverage'
```

Comment:
We should run both, because they behave differently

Reply:
Done